src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -747,6 +747,14 @@ class FileSegment(BaseFileSegment):
         allow_trailing=True,
     )
 
+    def get_table_references(self) -> Set[str]:
+        """Use parsed tree to extract table references."""
+        references = set()
+        for stmt in self.get_children("statement"):
+            stmt = cast(StatementSegment, stmt)
+            references |= stmt.get_table_references()
+        return references
+
```

Comment:
Is there a reason this is in this PR? Did I miss something?

Reply:
Ah - it's just a _move_. It was on `BaseFileSegment` and now it's on `FileSegment` in the ansi dialect, so that we the definition file for `BaseFileSegment` doesn't need to import `ansi_dialect`.